### PR TITLE
Add white background to paypal logo

### PR DESCRIPTION
**Pull request by rjzheng**
Add white background to the "Powered by PayPal" logo to resolve logo visibility issues.

## Before
Screen Shot 2021-01-25 at 10 33 47 AM

before 2

## After
Screen Shot 2021-01-25 at 12 30 04 PM
Screen Shot 2021-01-25 at 12 30 14 PM

### CODE REVIEW

File: src/ui/buttons/buttons.jsx [hunk 1]
```diff
@@ -170,19 +170,15 @@ export function Buttons(props : ButtonsProps) : ElementNode {
...
-
-            {
-                (layout === BUTTON_LAYOUT.VERTICAL && fundingSources.indexOf(FUNDING.CARD) !== -1)
-                    ? <PoweredByPayPal
-                        locale={ locale }
-                    /> : null
-            }
+            
+            {fundingSources.indexOf(FUNDING.CARD) !== -1 ? (
+                <div class="card-fields-with-logo">
```
Comment (gregjopa) [inline src/ui/buttons/buttons.jsx:175]: Please add `card-fields-with-logo` and `card-fields-container` to the [class constants file](https://github.com/paypal/paypal-checkout-components/blob/master/src/constants/class.js)

Comment (rjzheng) [reply src/ui/buttons/buttons.jsx:175]: Sure

**Comment by gregjopa** [comment]
Please hold off on reviewing this PR. @rjzheng is syncing up with the design team to ensure we support dark backgrounds correctly.

**Comment by bluepnume** [comment]
This has the potential to look pretty weird on non-white light backgrounds.

What if we:

- Add a dark-mode param (e.g. `style: { dark: true }`)
- When in dark mode, change the 'powered by PayPal' text and logo to a white/light color

**Comment by rjzheng** [comment]
> This has the potential to look pretty weird on non-white light backgrounds.
> 
> What if we:
> 
> * Add a dark-mode param (e.g. `style: { dark: true }`)
> * When in dark mode, change the 'powered by PayPal' text and logo to a white/light color

That's a great suggestion @bluepnume, though this would be configured by the merchant, so we can't guarantee the logo visibility. I believe the design team is figuring out the best way to guarantee that without any actions from the merchant.

**Comment by gregjopa** [comment]
Closing this PR out. A new one can be opened when the design team provides the specs.

**Pull request closed without merging**